ckan/lib/helpers.py

```diff
@@ -2344,6 +2344,35 @@ def mail_to(email_address, name):
     return html
 
 
+@core_helper
+def get_base_public_folder(app_conf):
```

Comment:
looks great, thanks! Just trying to keep lib/helpers from becoming a dumping ground for every utility function in ckan.